src/components/List/List.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const List = () => {
+  return (
+    <div>
```

Comment:
[Same here](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/pull/17/files#r785063332). About `main` tag. Please, I would appreciate it so much. 🙏